res/app/components/stf/install/install-service.js

```diff
@@ -89,12 +90,13 @@ module.exports = function InstallService(
 
   installService.installFile = function(control, $files) {
     var installation = new Installation('uploading')
+    var isIOSPlatform = AppState.device.platform === 'iOS'
```

Comment:
Is it necessary to use "var" instead of "let"?
"let" will scope the variable into this function (it won't be added to global scope)

Reply:
done